gridt/src/app/app.component.html

```diff
@@ -1,7 +1,7 @@
 <ion-app>
 
-    <ion-menu contentId="main" side="start" menuId="first">
-      <ion-header >
+    <ion-menu contentId="main" side="start" menuId="first" *ngIf="isLoggedIn$|async">
```

Comment:
Same for the pipe character. Readability is crucial, especially in the HTML, where code often is just less pretty.